include/lsst/afw/geom/SkyWcs.h

```diff
@@ -619,9 +628,7 @@ std::shared_ptr<TransformPoint2ToPoint2> getPixelToIntermediateWorldCoords(SkyWc
                                                                            bool simplify = true);
 
 /**
- * Print a SkyWcs to an ostream
- *
- * For now it just prints "SkyWcs"; eventually it would be nice to have a summary
+ * Print a SkyWcs to an ostream (delegates to SkyWcs.toString()).
```

Comment:
This won't link correctly. You want `SkyWcs::toString` (with or without parentheses).

Reply:
Huh. Never mind, [it works](http://doxygen.lsst.codes/stack/doxygen/x_masterDoxyDoc/namespacelsst_1_1afw_1_1geom.html#a9386e6373e2f43d46d287e8a64867ad2). Undocumented feature in Doxygen?